### PR TITLE
Update the munger to 0.4

### PR DESCRIPTION
**Pull request by brendandburns**
@thockin @eparis

### CODE REVIEW

File: mungegithub/run.sh [hunk 1]
```diff
@@ -16,7 +16,7 @@
 
 while true; do
     /mungegithub \
-      --pr-mungers=blunderbuss,lgtm-after-commit,remove-needs-rebase \
```
Comment (thockin) [inline mungegithub/run.sh]: did you mean to remove lgtm-after-commit ?

Comment (brendandburns) [inline mungegithub/run.sh]: yes, see: https://github.com/kubernetes/contrib/pull/45

Comment (thockin) [inline mungegithub/run.sh]: What's the correlation?  I'm missing somethign

Comment (brendandburns) [inline mungegithub/run.sh]: We weren't searching enough events, so we removed for some edge cases what
we should not have...

Comment (thockin) [inline mungegithub/run.sh]: I can't quite parse that sentence, but if you say it is intentional, I buy it.

**Comment by thockin** [comment]
LGTM

File: mungegithub/run.sh (hunk 1 above)
Comment (brendandburns) [inline mungegithub/run.sh]: silly phone auto-complete...

We weren't searching all of the events, so we didn't find an LGTM when it existed and we would spuriously remove LGTMs...

**Pull request merged**